### PR TITLE
community: Add bio for Teodor Duțu

### PR DESCRIPTION
**Pull request by teodutu**

### CODE REVIEW

**Review by lizababu — APPROVED** [review]
LGTM

File: website/docs/community/people.md [hunk 1]
```diff
@@ -4,4 +4,14 @@ title: People
...
 
-This is content
+## Teodor Duțu
+
+![Teodor Duțu](../../static/img/people/teodor_dutu.jpg)
+
+Teodor Duțu is a contributor to the OpenEdu project (21-COP-0016).
+He is developing content for the [Operating Systems class](https://github.com/systems-cs-pub-ro/operating-systems-oer/) and the [Security Summer School](https://github.com/security-summer-school/), as well as the [D Summer School](https://dlang-upb.github.io/D-Summer-School/).
+
+Teodor is a Masters student at the University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.
```
Comment (lizababu) [inline website/docs/community/people.md]:
```suggestion
Teodor is a Masters student at University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.
```

File: website/docs/community/people.md [hunk 2]
```diff
@@ -4,4 +4,14 @@ title: People
...
+He is also a teaching assistant at the University POLITEHNICA of Bucharest, being involved in the operating systems, data structures and hardware-software interface classes.
```
Comment (lizababu) [inline website/docs/community/people.md]:
```suggestion
He is also a teaching assistant at University POLITEHNICA of Bucharest, being involved in the Operating Systems, Data Structures and Hardware-Software Interface classes.
```

**Review by lizababu — CHANGES_REQUESTED** [review]

File: website/docs/community/people.md (hunk 1 above)
Comment (teodutu) [reply website/docs/community/people.md]: Are you sure? If it was just "University of Bucharest", there would definitely need to be a "the" before. Shouldn't it be "the POLITEHNICA University of Bucharest"? It sounds more natural to me like this.

Comment (lizababu) [reply website/docs/community/people.md]: `the POLITEHNICA University of Bucharest` sounds most natural to me, I'm not sure what the official name in english is for UPB. Maybe @razvand can give us the truth.

Comment (razvand) [reply website/docs/community/people.md]: I tend to not use `the`.

Comment (razvand) [reply website/docs/community/people.md]: But let's settle on something, and be consistent.

Comment (lizababu) [reply website/docs/community/people.md]: I updated my PR and I didn't use `the` there.

File: website/docs/community/people.md [hunk 3]
```diff
@@ -4,4 +4,15 @@ title: People
...
-This is content
+## Teodor Duțu
+
+![Teodor Duțu](../../static/img/people/teodor_dutu.jpg)
+
+I am a contributor to the OpenEdu project (21-COP-0016).
+I am developing content for the [Operating Systems class](https://github.com/systems-cs-pub-ro/operating-systems-oer/), the [Security Summer School](https://github.com/security-summer-school/), as well as the [D Summer School](https://dlang-upb.github.io/D-Summer-School/).
+
+I am a Masters student at University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.
+I am also a teaching assistant at the University POLITEHNICA of Bucharest, being involved in the Operating Systems, Data Structures and Hardware-Software Interface classes.
```
Comment (lizababu) [inline website/docs/community/people.md]:
```suggestion
I am also a teaching assistant at University POLITEHNICA of Bucharest, being involved in the Operating Systems, Data Structures and Hardware-Software Interface classes.
```

**Review by lizababu — APPROVED** [review]

File: website/docs/community/people.md [hunk 4]
```diff
@@ -4,4 +4,15 @@ title: People
...
+![Teodor Duțu](../../static/img/people/teodor_dutu.jpg)
+
+I am a contributor to the OpenEdu project (21-COP-0016).
+I am developing content for the [Operating Systems class](https://github.com/systems-cs-pub-ro/operating-systems-oer/), the [Security Summer School](https://github.com/security-summer-school/), as well as the [D Summer School](https://dlang-upb.github.io/D-Summer-School/).
+
+I am a Masters student at University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.
+I am also a teaching assistant at University POLITEHNICA of Bucharest, being involved in the Operating Systems, Data Structures and Hardware-Software Interface classes.
+I am interested in compilers, programming language analysis, operating systems, security and embedded programming and passionate about open source and I am also a contributor to the [D programming language](https://dlang.org/).
+
+Apart from computer science, I like history, philisophy, literature, healthy cooking and running.
```
Comment (razvand) [inline website/docs/community/people.md]:
```suggestion
Apart from computer science, I like history, philosophy, literature, healthy cooking and running.
```

File: website/docs/community/people.md [hunk 5]
```diff
@@ -4,4 +4,15 @@ title: People
...
-This is content
+## Teodor Duțu
+
+![Teodor Duțu](../../static/img/people/teodor_dutu.jpg)
+
+I am a contributor to the OpenEdu project (21-COP-0016).
+I am developing content for the [Operating Systems class](https://github.com/systems-cs-pub-ro/operating-systems-oer/), the [Security Summer School](https://github.com/security-summer-school/), as well as the [D Summer School](https://dlang-upb.github.io/D-Summer-School/).
+
+I am a Masters student at University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.
+I am also a teaching assistant at University POLITEHNICA of Bucharest, being involved in the Operating Systems, Data Structures and Hardware-Software Interface classes.
```
Comment (razvand) [inline website/docs/community/people.md]:
```suggestion
I am also part of teaching teams at University POLITEHNICA of Bucharest, being involved in the Operating Systems, the Data Structures and the Hardware-Software Interface classes.
```

**Review by razvand — CHANGES_REQUESTED** [review]
Some minor comments.

File: website/docs/community/people.md [hunk 6]
```diff
@@ -19,4 +19,17 @@ I love spending time with students, teaching and finding ways to improve the cur
 
 I currently work at [Bitdefender](https://www.bitdefender.com/) as a software engineer, part of the [Connected Home Security](https://www.bitdefender.com/smart-home/) team.
 
-Apart from computer science, I really enjoy skiing, dancing and cooking.
\ No newline at end of file
+Apart from computer science, I really enjoy skiing, dancing and cooking.
```
Comment (razvand) [inline website/docs/community/people.md]: Why is this line here as modified?

Comment (razvand) [reply website/docs/community/people.md]: Is it because @lizababu didn't end her commit with a newline?

**Review by razvand — CHANGES_REQUESTED** [review]
Please rebase your commit / PR.

File: website/docs/community/people.md (hunk 6 above)
Comment (teodutu) [reply website/docs/community/people.md]: Yes.

**Comment by teodutu** [comment]
@razvand done.

**Review by razvand — CHANGES_REQUESTED** [review]
Please rebase the PR to only have a commit on top of `main`.

Resize your image to about 200-250 size. See the syntax [here](https://raw.githubusercontent.com/open-education-hub/open-education-hub-site/main/website/docs/community/people.md).

**Review by razvand — APPROVED** [review]
All good. Thanks, @teodutu .
